public/app/features/teams/TeamMembers.test.tsx

```diff
@@ -27,34 +51,92 @@ const setup = (propOverrides?: object) => {
 };
 
 describe('Render', () => {
+  beforeEach(() => {
```

Comment:
Maybe change to afterEach?
```suggestion
  afterEach(() => {
```

Reply:
Refactored away this